### PR TITLE
External address VPC resource

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

**Review by ksamoray — APPROVED** [review]
No way to test this as we need a VM/port?

File: nsxt/resource_nsxt_vpc_external_address.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+		Read:   resourceNsxtVpcExternalAddressRead,
+		Update: resourceNsxtVpcExternalAddressUpdate,
+		Delete: resourceNsxtVpcExternalAddressDelete,
+		Importer: &schema.ResourceImporter{
+			State: nsxtParentPathResourceImporter,
+		},
+		Schema: map[string]*schema.Schema{
+			"parent_path":                getPolicyPathSchema(true, true, "port path for address binding"),
+			"allocated_external_ip_path": getPolicyPathSchema(true, false, "allocated ip path"),
+			"external_ip_address": {
```
Comment (salv-orlando) [inline nsxt/resource_nsxt_vpc_external_address.go:24]: Did we check if this is actually computed, or can instead be used to provide an IP address not allocated from a block? (ie: a random IP selected by the user)

Comment (annakhm) [reply nsxt/resource_nsxt_vpc_external_address.go:24]: Its marked as `read-only` in the spec

**Comment by annakhm** [comment]
> No way to test this as we need a VM/port?

We can test this with predefined environment variable. I will post the test in a separate PR

**Pull request merged**